### ISSUE TITLE
Document hidden hotkeys + settings file options

### ISSUE THREAD

**Issue by jamespharvey20** [sublimehq/sublime_merge#1369]
**Problem description**

It appears there are several options not easily discoverable:
* [press CTRL+8 to toggle single branch](https://github.com/sublimehq/sublime_merge/issues/914)
* [in setting file, add option `ui_scale` that isn't in the file by default or in preferences ](https://github.com/sublimehq/sublime_merge/issues/851)

**Preferred solution**

Either documentation should be added to list all hotkeys and document the settings file, or both.

My apologies if I've overlooked something, other than looking through github issues and the Sublime forum.

**Comment by Ultra-Instinct-05** [comment]
`ui_scale` is a ST setting. Since ST & SM have the same foundational base, many settings in ST may work out of the box in SM,

**Comment by RobertPaauw** [comment]
I agree this should be easier to find and/or better documented. The good news is that there are some files that list these settings and hotkeys.

The default settings can be found inside `Default - Merge.sublime-package` and `Default.sublime-package` in the Sublime Merge instal directory (`C:\Program Files\Sublime Merge\Packages` on my machine).
The `.sublime-package` can be unzipped with something like 7zip.

From [#166](https://github.com/sublimehq/sublime_merge/issues/166#issuecomment-433112935):
>... the default Preferences in SM are aggregated from 3 files:
> 
> - Default/Preferences.sublime-settings
> - Default/Preferences ().sublime-settings
> - Default - Merge/Preferences.sublime-settings

Hotkeys can be found in the `*.sublime-keymap` files in the same packages (although without any comments on what they do).
